c++/src/kj/compat/http.c++

```diff
@@ -1805,15 +1803,16 @@ private:
 
   Promise<size_t> tryReadInternal(void* buffer, size_t minBytes, size_t maxBytes,
```

Comment:
Since the only caller of this is now `tryRead()` itself, can we inline it?

Reply:
(Preferably as a separate commit to be able to see changes more clearly.)

Reply:
Hmm, I screwed up the "separate commit" part, but I think the diff looks okay.